docs/2_auth.md

```diff
@@ -85,6 +86,33 @@ Note: The user is checked against the group members list on initial authenticati
 
 Note: When using the Azure Auth provider with nginx and the cookie session store you may find the cookie is too large and doesn't get passed through correctly. Increasing the proxy_buffer_size in nginx or implementing the [redis session storage](configuration#redis-storage) should resolve this.
 
+### Azure OIDC Auth Provider
+
+1. Go to [https://portal.azure.com](https://portal.azure.com), and create new "App Registration".
```

Comment:
All of the entries in this list are numbered `1`, can we number them as a list please